Crane/app/src/androidTest/java/androidx/compose/samples/crane/details/DetailsActivityTest.kt

```diff
@@ -59,61 +51,67 @@ class DetailsActivityTest {
     var hiltRule = HiltAndroidRule(this)
 
     @get:Rule(order = 1)
-    val composeTestRule = createEmptyComposeRule()
+    val composeTestRule = createComposeRule()
+
+    private lateinit var mapState: CameraPositionState
 
     @Before
     fun setUp() {
         hiltRule.inject()
+
         cityDetails = destinationsRepository.getDestination(MADRID.name)!!
-        scenario = ActivityScenario.launch(
-            createDetailsActivityIntent(
-                InstrumentationRegistry.getInstrumentation().targetContext,
-                testExploreModel
-            )
-        )
-    }
+        mapState = CameraPositionState()
+
+        val countDownLatch = CountDownLatch(1)
+
+        composeTestRule.setContent {
+            CraneTheme {
+                Column {
```

Comment:
This `Column` shouldn't be needed if added to the `MapViewContainer`, see my comment below